src/ClassicUO.Client/LegionScripting/API.py

```
@@ -24,293 +24,1376 @@ class Notoriety:
+    
+     textbox = API.CreateGumpTextBox("Text example", w, h, True)
+    
+     gump.Add(textbox)
+    
+     API.AddGump(gump)
+     ```
+    
+    """
     pass
 
 def CreateGumpTTFLabel(text: str, size: float, color: str = "#FFFFFF", font: str = TrueTypeLoader.EMBEDDED_FONT, aligned: str = "let", maxWidth: int = 0, applyStroke: bool = False):
```

Ruff (0.11.9)

Undefined name `TrueTypeLoader`

(F821)